content/replication.go

```diff
@@ -205,8 +205,14 @@ func (cm *ContentManager) ensureStorage(ctx context.Context, content util.Conten
 
 	// after reconciling content deals,
 	// check If this is a shuttle content and that the shuttle is online and can start data transfer
-	if content.Location != constants.ContentLocationLocal && !cm.shuttleMgr.IsOnline(content.Location) {
-		cm.log.Warnf("content shuttle: %s, is not online for deal making", content.Location)
+	isOnline, err := cm.shuttleMgr.IsOnline(content.Location)
+	if err != nil {
+		done(time.Minute * 15)
```

Comment:
Not sure what 15 minutes is, and looks like we use it below as well. Should we extract it as a variable?

Reply:
lol, this part of content manager will be extracted to a different package. That timing will not be used anymore. I did not just added it, so I don't want to modify it